app/src/main/java/com/worldsnas/starplayer/view/PlayerFragment.kt

```diff
@@ -33,23 +31,21 @@ class PlayerFragment : Fragment() {
         // Inflate the layout for this fragment
         viewBinding = FragmentPlayerBinding.inflate(inflater, container, false)
         initializeExoplayer()
+        setMusicArguments()
         return viewBinding?.root
     }
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
-
-        playerComponent = DaggerPlayerComponent.builder()
-            .appComponent((activity!!.application as App).appComponent).build()
-        playerComponent.inject(this)
+        daggerSetup()
     }
 
     private fun initializeExoplayer() {
         exoPlayer = SimpleExoPlayer.Builder(context!!).build()
         viewBinding?.exoControlView?.player = exoPlayer
 
-        val filePath = Environment.getExternalStorageDirectory().path + "/Download/Temp/Test.mp3"
-        val mediaSource = buildMediaSource(Uri.fromFile(File(filePath)))
+        val filePath = "content://media/" + arguments?.getString("address")
```

Comment:
che jaleb in bayad ezafe she ta kar kone?
bedone `content` nemishe?
ajibe

Reply:
kheir moteasefane nemishe, behtar nist ye variable tarif konam barash?

Reply:
na bashe felan
vali const bashe behtare